multi-node/aws/artifacts/scripts/install-controller.sh

```diff
@@ -111,17 +108,17 @@ EOF
 	}
 
 	mkdir -p /etc/kubernetes/manifests
-	template manifests/controller/kube-proxy.yaml /etc/kubernetes/manifests/kube-proxy.yaml
-	template manifests/controller/kube-apiserver.yaml /etc/kubernetes/manifests/kube-apiserver.yaml
-	template manifests/controller/kube-podmaster.yaml /etc/kubernetes/manifests/kube-podmaster.yaml
+	template /tmp/manifests/controller/kube-proxy.yaml /etc/kubernetes/manifests/kube-proxy.yaml
+	template /tmp/manifests/controller/kube-apiserver.yaml /etc/kubernetes/manifests/kube-apiserver.yaml
+	template /tmp/manifests/controller/kube-podmaster.yaml /etc/kubernetes/manifests/kube-podmaster.yaml
```

Comment:
Can we namespace this? Maybe "/tmp/coreos/manifests/..." Or even just switch $ARTIFACT_URL to $ARTIFACT_DIR to allow a user to easily override..